### PR TITLE
GitHub actions and jest

### PR DESCRIPTION
**Pull request by nicolasbrugneaux**
Inspired by #3 but modernized. Will add a similar test case

Note: Looks like a lot of changes because it's based on the following PR:
- #13 

I will rebase it when it's merged.

### CODE REVIEW

**Comment by nicolasbrugneaux** [comment]
Now that #13 was merged, I'll tackle this on Monday, can I ask you to setup an `process.env.MNEMONIC` for this repo (or give me permission to add it? I think `process.env.ACCOUNT` should be refactored to not be needed

File: yarn.lock [hunk 1]
```diff
@@ -12,171 +12,3631 @@ __metadata:
   languageName: node
   linkType: hard
 
+"@ampproject/remapping@npm:^2.2.0":
```
[the remark this answers is not in the document]
Comment (nicolasbrugneaux) [reply yarn.lock:15]: I know!!

File: src/lib/StaticCeloProvider.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { JsonRpcApiProvider } from "ethers";
```
[the remark this answers is not in the document]
Comment (nicolasbrugneaux) [reply src/lib/StaticCeloProvider.ts:1]: Oops

**Comment by nicolasbrugneaux** [comment]
Tests are passing on the celo-org fork: https://github.com/celo-org/celo-ethers-wrapper/actions/runs/6629109115/job/18007671031

**Review by jmrossy — APPROVED** [review]

**Pull request merged**